src/field_theory/ratfunc.lean

```diff
@@ -473,7 +482,7 @@ variables {K}
 section lift_hom
 
 variables {G₀ L R S F : Type*} [comm_group_with_zero G₀] [field L] [comm_ring R] [comm_ring S]
-omit hring
+--omit hring  -- pre-porting note: a little scary, but maybe innocuous: no `K` is present here.
```

Comment:
I think this is safe to delete since there is no `hring` defined in this file anymore.
```suggestion
```